### PR TITLE
improvements in sign up and registration screens

### PR DESCRIPTION
**Pull request by ranisa-gupta16**
# feat/fix/chore: pr title

## JIRA Ticket

https://jira.csiro.au/browse/BSS-1301

## Description

Will need to basically test in Emulator how its looking, if anything more can be improved, tested it in mozilla + edge responsive mobile screens and looks fine. 

This PR makes them comfortably readable on Android without pinch-zoom, 
stops iOS from auto-zooming on input focus, and tidies up the password 
strength meter.

image

## Checklist

- [ ] I have confirmed all commits have been signed.
- [ ] I have added JSDoc style comments to any new functions or classes.
- [ ] Relevant documentation such as READMEs, guides, and class comments are updated.

### CODE REVIEW

File: api/public/password-strength.css [hunk 1]
```diff
@@ -9,11 +9,14 @@
 
 .strength-bar-container {
   width: 100%;
-  height: 8px;
+  /* Slightly taller bar reads better on mobile without dominating the form. */
+  height: 10px;
   background-color: #e9ecef;
-  border-radius: 4px;
+  border-radius: 999px;
```
Comment (PeterBaker0) [inline api/public/password-strength.css]: What's the go with the huge border radius?

**Review by PeterBaker0 — APPROVED** [review]
Does make larger on phone, checked and looked fine to me. 

Question about removing of the crack time + the huge border radius.

File: api/public/password-strength.css (hunk 1 above)
Comment (ranisa-gupta16) [reply api/public/password-strength.css]: yeah it was pill approach but all sorted now.
